packages/common-lib/src/services/worksheetRegistryService.js

```diff
@@ -0,0 +1,182 @@
+import mapInterfaceData from './mapInterfaceData'
```

Comment:
can we move this to worksheet module?

Reply:
this approach suggested by @ChakshuGautam